engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -68,9 +71,7 @@ public KitchenCommandExecutor( Class<?> pkgClazz, LogChannelInterface log, Futur
     setKettleInit( kettleInit );
   }
 
-  public Result execute( String repoName, String noRepo, String username, String trustUser, String password, String dirName,
-                         String filename, String jobName, String listJobs, String listDirs, String exportRepo, String initialDir,
-                         String listRepos, String listParams, NamedParams params, NamedParams customParams, String[] arguments ) throws Throwable {
+  public Result execute( JobParams iParams ) throws Throwable {
```

Comment:
**Nitpick**: can we rename `iParams` to just `params` ? It's an odd name for a java variable...

Reply:
Offcourse..